File: threedollar-domain/src/main/java/com/depromeet/threedollar/domain/domain/storeimage/StoreImageCreator.java
package com.depromeet.threedollar.domain.domain.storeimage;

import com.depromeet.threedollar.common.docs.ObjectMother;
import com.depromeet.threedollar.domain.domain.store.Store;
import lombok.AccessLevel;
import lombok.NoArgsConstructor;

@ObjectMother
@NoArgsConstructor(access = AccessLevel.PRIVATE)
class StoreImageCreator {

    static StoreImage create(Store store, Long userId, String url) {
        return StoreImage.builder()
            .store(store)
            .userId(userId)
            .url(url)
            .build();
    }

}

File: threedollar-domain/src/testFixtures/java/com/depromeet/threedollar/domain/user/domain/storeimage/StoreImageCreator.kt
package com.depromeet.threedollar.domain.user.domain.storeimage

import com.depromeet.threedollar.domain.user.domain.ObjectMother
import com.depromeet.threedollar.domain.user.domain.store.Store

@ObjectMother
object StoreImageCreator {

    @JvmStatic
    fun create(
        store: Store,
        userId: Long,
        url: String
    ): StoreImage {
        return StoreImage.builder()
            .store(store)
            .userId(userId)
            .url(url)
            .build()
    }

}
